doc/source/rllib.rst

```diff
@@ -7,155 +7,103 @@ RLlib is an open-source library for reinforcement learning that offers both high
 
 .. code-block:: bash
 
-  pip install tensorflow  # or tensorflow-gpu
   pip install ray[rllib]  # also recommended: ray[debug]
 
-You might also want to clone the `Ray repo <https://github.com/ray-project/ray>`__ for convenient access to RLlib helper scripts:
+Then, you can try out training in the following equivalent ways:
 
 .. code-block:: bash
 
-  git clone https://github.com/ray-project/ray
-  cd ray/rllib
-
-Training APIs
--------------
-* `Command-line <rllib-training.html>`__
-* `Configuration <rllib-training.html#configuration>`__
-* `Python API <rllib-training.html#python-api>`__
-* `Debugging <rllib-training.html#debugging>`__
-* `REST API <rllib-training.html#rest-api>`__
-
-Environments
-------------
-* `RLlib Environments Overview <rllib-env.html>`__
-* `Feature Compatibility Matrix <rllib-env.html#feature-compatibility-matrix>`__
-* `OpenAI Gym <rllib-env.html#openai-gym>`__
-* `Vectorized <rllib-env.html#vectorized>`__
-* `Multi-Agent and Hierarchical <rllib-env.html#multi-agent-and-hierarchical>`__
-* `Interfacing with External Agents <rllib-env.html#interfacing-with-external-agents>`__
-* `Advanced Integrations <rllib-env.html#advanced-integrations>`__
-
-Models, Preprocessors, and Action Distributions
------------------------------------------------
-* `RLlib Models, Preprocessors, and Action Distributions Overview <rllib-models.html>`__
-* `TensorFlow Models <rllib-models.html#tensorflow-models>`__
-* `PyTorch Models <rllib-models.html#pytorch-models>`__
-* `Custom Preprocessors <rllib-models.html#custom-preprocessors>`__
-* `Custom Action Distributions <rllib-models.html#custom-action-distributions>`__
-* `Supervised Model Losses <rllib-models.html#supervised-model-losses>`__
-* `Variable-length / Parametric Action Spaces <rllib-models.html#variable-length-parametric-action-spaces>`__
-* `Autoregressive Action Distributions <rllib-models.html#autoregressive-action-distributions>`__
-
-Algorithms
-----------
-
-*  High-throughput architectures
-
-   -  `Distributed Prioritized Experience Replay (Ape-X) <rllib-algorithms.html#distributed-prioritized-experience-replay-ape-x>`__
-
-   -  `Importance Weighted Actor-Learner Architecture (IMPALA) <rllib-algorithms.html#importance-weighted-actor-learner-architecture-impala>`__
-
-   -  `Asynchronous Proximal Policy Optimization (APPO) <rllib-algorithms.html#asynchronous-proximal-policy-optimization-appo>`__
+  rllib train --run=PPO --env=CartPole-v0
 
-*  Gradient-based
+.. code-block:: python
 
-   -  `Advantage Actor-Critic (A2C, A3C) <rllib-algorithms.html#advantage-actor-critic-a2c-a3c>`__
+  from ray import tune
+  from ray.rllib.agents.ppo import PPOTrainer
+  tune.run(PPOTrainer, config={"env": "CartPole-v0"})
 
-   -  `Deep Deterministic Policy Gradients (DDPG, TD3) <rllib-algorithms.html#deep-deterministic-policy-gradients-ddpg-td3>`__
+Policies
```

Comment:
Maybe right before this mention here the three concepts you'll cover: Policies, Samples, and Trainers.